test_home.py

```diff
@@ -54,6 +54,30 @@ def test_more_dropdown_navigates_correctly(base_url, selenium, i, page_url):
     assert page_url in selenium.current_url
 
 
+@pytest.mark.parametrize(
+    'count, category',
+    enumerate([
+        'Abstract',
+        'Nature',
+        'Film',
+        'Scenery',
+        'Music',
+        'Seasonal',
+    ])
+)
+@pytest.mark.nondestructive
+@pytest.mark.desktop_only
+def test_theme_categories_shelf(base_url, selenium, count, category):
+    page = Home(selenium, base_url).open()
```

Comment:
Could we add a comment here with a brief explanation?

Reply:
I've added two comments to make it more clear what this test is about